### PR TITLE
Monster lookups

### PR DESCRIPTION
**Pull request by RubenJ01**
Looks up monsters from the OSRS Box module.
Added this dependency to the readme.

### CODE REVIEW

File: cogs/links.py
Comment (RubenJ01) [inline cogs/links.py]: Just realised that this isnt very clean code.
It needs to be drapped in to and if else block and the code can be shortened to 2 lines with list comprehension.

**Comment by zedchance** [comment]
Hi I'm viewing the changes now. Heres some things I've noticed:

- [x] The gp amounts should be formatted with commas (i.e. 15,000 gp)
- [x] Capitalize the weakness/attack styles

Is there anything else interesting in the response from osrsbox thats worth display? Is there an url to an icon?

Good work!

EDIT: Also what IDE are you using? Did you run an auto-format on the file `links.py`?

**Comment by RubenJ01** [comment]
Im using pycharm, it yelled at all the code and i reformatted it. You should take a look at pep8.

Il make some adjustments, i think the wiki link is worth displaying but not much more.

**Comment by RubenJ01** [comment]
I just noted that i reformatted `main.py` this should have no effect on its functionality.

**Comment by zedchance** [comment]
Nice work. I love how fast the command is!

One last thing before I merge, can you format the drops with a comma and remove the underline?

Ex. 1/3,000 instead of 1/3000

**Comment by RubenJ01** [comment]
Sure, il push the changes tonight

File: helpers/monsters.py
Comment (zedchance) [inline helpers/monsters.py:22]: You might consider playing around with letting this search for substrings also. For example in the ge command I have you can search `dragon pickaxe` or `dragon pick` and they both come up. This is accomplished like this:
https://github.com/zedchance/blues_bot.py/blob/c5866d86c443b4b1f271c97d2b24b71291dabebd/helpers/ge.py#L22
If you want to include this change in this PR or to address it after is up to you, but I think its definitely worth messing around with. (Definitely test it because sometimes it can have some unintended consequences.)

Comment (RubenJ01) [reply helpers/monsters.py:22]: This seems very helpfull, definetely something il include within the next commit

**Review by zedchance — CHANGES_REQUESTED** [review]
One last thing I notice:
Screen Shot 2020-01-07 at 1 50 17 PM
This should be a little more user friendly on what went wrong.

**Comment by RubenJ01** [comment]
> One last thing I notice:
> Screen Shot 2020-01-07 at 1 50 17 PM
> This should be a little more user friendly on what went wrong.

Doesnt the error handler deal with this?

**Pull request merged**
